src/cloud/project/services-redis.md

```diff
@@ -63,3 +63,26 @@ Assuming your Redis relationship is named `redis`, you can access it using the `
    ```bash
    redis-cli -h redis.internal
    ```
+
+## Get installed Redis version
+
+Use the following command to get a Redis version on the Integration environment:
```

Comment:
```suggestion
Use the following command to get the Redis version installed in an Integration environment:
```

Reply:
**done**

Reply:
Thank you, @meker12

Reply:
Normally we'd say "ON an Integration environment"

Reply:
Same goes for above on the Staging & Production

Reply:
**done**

@BarnyShergold , could you please review?

Thank you!